src/main/scala/ch/epfl/bluebrain/nexus/sbt/nexus/ReleasePlugin.scala

```diff
@@ -30,6 +30,7 @@ object ReleasePlugin extends AutoPlugin {
       val v = version.value
       val major = v.split("\\.")(0)
       val minor = v.split("\\.")(1)
+      val contributors = ("git shortlog -sne HEAD" #| "cut -f2" !!) split "\n"
```

Comment:
does this command get all contributors of this specific release, or of the entire project?

Reply:
it gets contributors of the entire project